ppanggolin/workflow/workflow.py

```diff
@@ -28,20 +28,26 @@ def launch(args):
             getSeq = False
         readAnnotations(pangenome, args.anno, getSeq)
         writePangenome(pangenome, filename, args.force)
-        if args.clusters is None and pangenome.status["geneSequences"] == "No" and args.fasta is None:
+        if args.clusters is None and pangenome.status["geneSequences"] == "No" and args.fasta_list is None and args.fasta_files is None:
             raise Exception("The gff/gbff provided did not have any sequence informations, you did not provide clusters and you did not provide fasta file. Thus, we do not have the information we need to continue the analysis.")
 
-        elif args.clusters is None and pangenome.status["geneSequences"] == "No" and args.fasta is not None:
-            getGeneSequencesFromFastas(pangenome, args.fasta)
+        elif args.clusters is None and pangenome.status["geneSequences"] == "No" and args.fasta_list is not None:
+            getGeneSequencesFromFastas(pangenome, args.fasta_list)
+
+        elif args.clusters is None and pangenome.status["geneSequences"] == "No" and args.fasta_files is not None:
+            getGeneSequencesFromFastas(pangenome, args.fasta_files)
```

Comment:
This is for the case where the user provided annotation through a file listing annotation files, and genome sequences through the --fasta_files option. Do we really want to allow this ?